packages/vue-component-library/src/components/RnTransfer/scripts.js

```diff
@@ -0,0 +1,39 @@
+export default {
+  name: 'RnTransfer',
+
+  props: {
+    listData: !Array,
+    leftHeader: String,
+    rightHeader: String
+  },
+
+  computed: {
+    leftList () {
+      return this.listData.filter(item => item.list === 'left')
+    },
+
+    rightList () {
+      return this.listData.filter(item => item.list === 'right')
+    }
+  },
+
+  methods: {
+    move (item, side) {
+      if (item.checked === true) {
+        // eslint-disable-next-line no-param-reassign
```

Comment:
Didn't we talk about this before? I thought the agreement was no mutation of params

Reply:
It feels like the most efficient way to do it though, the list data comes in like this:

```
    {
        "id": "stfaith",
        "location": 'St Faith', 
        "list" : 'left',
      },
```
and just changing `item.list` to `right` is a quick way that works well. ultimately what is returned is a mutated JSON object anyway so surely it doesn't really matter.

Reply:
Mutating data can have weird side effects, like causing infinite re-render. Can't we just return a new value?